src/rafimage.cpp

```diff
@@ -72,7 +72,7 @@ void RafImage::printStructure(std::ostream& out, PrintStructureOption option, si
   if (bPrint) {
     io_->seek(0, BasicIo::beg);  // rewind
     size_t address = io_->tell();
-    constexpr auto format = " %9zu | %9" PRIu32 " | ";
+    constexpr auto format = " {:8} | {:8} | ";
```

Comment:
```suggestion
    constexpr auto format = " {:9} | {:9} | ";
```

Reply:
IIRC, I reduced it because of failing tests.

Reply:
Hm, no tests were failing when I made it wider recently in https://github.com/Exiv2/exiv2/pull/2824, so I think there is no test that covers `-pS` on RAF files, which are now easily getting over 100MB (50Mpix and 100MPix models).

Reply:
Made the change.